src/test/java/OrderTestParametrized.java

```diff
@@ -0,0 +1,138 @@
+import pageObjects.ApprovalOrderPage;
+import pageObjects.MainPage;
+import pageObjects.OrderPage;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+@RunWith(Parameterized.class)
+public class OrderTestParametrized {
+
+    private WebDriver driver;
+    private final String name;
+    private final String lastname;
+    private final String address;
+    private final String numberPhone;
+    private final String comment;
+    private final String station;
+    private final String data;
+    private final String day;
+
+    public OrderTestParametrized(String name, String lastname, String address, String numberPhone, String comment, String station, String data, String day) {
+        this.name = name;
+        this.lastname = lastname;
+        this.address = address;
+        this.numberPhone = numberPhone;
+        this.comment = comment;
+        this.station = station;
+        this.data = data;
+        this.day = day;
+    }
+
+    @Parameters
+    public static String[][] getData () {
+        return new String[][] {
+                {"Петр", "Петров", "Невский проспект 10", "+79111111111", "Комментарий 1", "Сокольники", "01.04.2023\n", "сутки"},
+                {"Иван", "Иванов", "Ленина 67", "89117409387", "Комментарий 2", "Лубянка", "05.05.2023\n", "двое суток"}
+        };
+    }
+
+    @Before
+    public void setUp() {
+        Properties prop = new Properties();
+        try {
+            prop.load(new FileReader("application.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        String webDriver = prop.getProperty("webdriver");
+        if (webDriver.equals("firefox")) {
+            FirefoxOptions options = new FirefoxOptions();
+            WebDriverManager.firefoxdriver().setup();
+            driver = new FirefoxDriver(options);
+        } else if (webDriver.equals("chrome")) {
+            WebDriverManager.chromedriver().setup();
+            driver = new ChromeDriver();
+        }
+        driver.get("https://qa-scooter.praktikum-services.ru/");
+    }
+
+    @Test
+    public void checkCreateNewOrder_withButtonTop_checkWindowAboutASuccessfulOrder() { //Позитивный кейс по формированию заказа по кнопке "Заказать" в шапке сайта
+
+        MainPage mainPage = new MainPage(driver);
+        OrderPage orderPage = new OrderPage(driver);
+        ApprovalOrderPage approvalOrderPage = new ApprovalOrderPage(driver);
+        AboutRentPage aboutRentPage = new AboutRentPage(driver);
+
+        mainPage.open();
+        mainPage.clickCreateOrderButtonInHead();
+        orderPage.setName(name);
+        orderPage.setLastname(lastname);
+        orderPage.setAddress(address);
+        orderPage.setNumberPhone(numberPhone);
+        orderPage.chooseMetroStation(station);
+        OrderPage.wait(1);
+        orderPage.clickNextButton();
+        OrderPage.wait(3);
+        aboutRentPage.setData(data);
+        aboutRentPage.chooseRentalDay(day);
```

Comment:
Это нужно обязательно исправить:

на этом шаге тест падает -- это нужно обязательно исправить

Reply:
Не очень понятно что значит "падает", у меня эти шаги корректно работают, что именно нужно исправить?

Reply:
Код стабильно падает на этом шаге. Ошибка:
org.openqa.selenium.ElementClickInterceptedException: Element <div class="Dropdown-placeholder"> is not clickable at point (640,261) because another element <div class="react-datepicker__current-month"> obscures it

Ошибка возникает потому что на предыдущем шаге aboutRentPage.setData(data); открывается календарь, затем он не закрывается -- это и стопит шаг aboutRentPage.chooseRentalDay(day);.

Можно с наставником обсудить этот момент -- он подскажет как быть

Reply:
Поправила